### DISCUSSION TITLE
how to model a circuit termination with multiple handoffs?

### DISCUSSION THREAD

**Discussion by ghost** [netbox-community/netbox#17494] [Q&A]
if I have a circuit and I want to set a 'termination z' for the circuit going towards my network, let's say an HA pair of firewalls.
The carrier is giving me 2 handoffs from their gear. How do I model a single circuit terminating on multiple pieces of equipment on my side? (firewall 1 and firewall 2)

**Comment by candlerb** [comment]
I think your choices are:

1. Model the provider's demarc box as a device in Netbox (like a switch). Have the circuit terminate on one port, and connect the other two ports to your firewalls. Or:
2. Make two circuits with their own terminations. If the provider's circuit ID is "foo" then call them "foo-port1" and "foo-port2". Or:
3. Make one circuit and connect it to the primary firewall. Mark the port on the second firewall as "connected".

**Comment by alehaa** [comment]
Could be the same problem as I encountered in https://github.com/netbox-community/netbox/discussions/16917.

If it‘s a physical device of the provider or a patch panel, just model it like an internal one and connect the rear port to a circuit. Otherwise NetBox doesn't provide a builtin solution and the temporary fixes mentioned above apply.
